Rectangle.cpp

```diff
@@ -1,15 +1,30 @@
 #ifndef Rectangle_cpp
 #define Rectangle_cpp
+
 #include <iostream>
-#include "Draw.cpp"
+#include "Shape.cpp"
+#include "Area.cpp"
+#include "Input.cpp"
+
 using namespace std;
 
-class Rectangle : public Draw
+class Rectangle :  public Shape , public Input
```

Comment:
Same as before `Input` implements `inputRadius` which has no use in `Rectangle`.